### PR TITLE
add lots of components i dont know which ones

### PR DESCRIPTION
**Pull request by DawidKapka**

### CODE REVIEW

File: src/app/ui/button/button.component.html [hunk 1]
```diff
@@ -7,9 +7,12 @@
...
+    'success-inverted': color === 'success-inverted',
     danger: color === 'danger',
-    error: color === 'error'
+    'danger-inverted': color === 'danger-inverted',
+    error: color === 'error',
+    'error-inverted': color === 'error-inverted'
   }"
 >
-  {{ value }}
+  <ng-content></ng-content>
```
Comment (DeNic0la) [inline src/app/ui/button/button.component.html:17]: POW

File: src/app/ui/button/button.component.ts [hunk 2]
```diff
@@ -7,7 +7,6 @@ import { ButtonColor, ButtonSize } from './button.types';
   styleUrls: ['./button.component.scss'],
 })
 export class ButtonComponent {
-  @Input('text') value: string = 'text';
   @Input('size') size: ButtonSize = 'medium';
   @Input('color') color: ButtonColor = 'primary';
```
Comment (DeNic0la) [inline src/app/ui/button/button.component.ts:11]: Definitely not a fan of the Input aliases

File: src/app/ui/icon/icon.component.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<img
+  [src]="'../../../assets/images/icons/' + name + '.svg'"
```
Comment (DeNic0la) [inline src/app/ui/icon/icon.component.html:2]: Is this working the way you want it to? I would prefer to have those .svg files declared as assets

**Review by DeNic0la — APPROVED** [review]
acceptable

File: src/app/ui/button/button.component.ts (hunk 2 above)
Comment (DawidKapka) [reply src/app/ui/button/button.component.ts:11]: what's wrong tho?

File: src/app/ui/icon/icon.component.html (hunk 3 above)
Comment (DawidKapka) [reply src/app/ui/icon/icon.component.html:2]: Yes, it is working the way I want it to.

File: src/app/ui/button/button.component.html (hunk 1 above)
Comment (DawidKapka) [reply src/app/ui/button/button.component.html:17]: ?

Comment (DeNic0la) [reply src/app/ui/button/button.component.html:17]: POW

Comment (DawidKapka) [reply src/app/ui/button/button.component.html:17]: POW

File: src/app/ui/button/button.component.ts (hunk 2 above)
Comment (DeNic0la) [reply src/app/ui/button/button.component.ts:11]: they are not needed in that case, there is nothing wrong tho, this is why i pressed the approve button

File: src/app/ui/icon/icon.component.html (hunk 3 above)
Comment (DeNic0la) [reply src/app/ui/icon/icon.component.html:2]: Ok

**Pull request merged**